eclide/EclDlgBuilder.cpp

```diff
@@ -81,7 +81,10 @@ bool CBuilderDlg::DoSave(bool attrOnly)
         IAttributeBookkeep attrProcessed;
         MetaInfo metaInfo;
         m_attribute->PreProcess(PREPROCESS_SAVE, NULL, attrs, attrProcessed, errors, metaInfo);
-        SendMessage(CWM_SUBMITDONE, Dali::WUActionCheck, (LPARAM)&errors);
+        if (m_attribute->GetType() != CreateIAttributeECLType() || m_attribute->GetType() == CreateIAttributeECLType() && !errors.empty())
+        {
+            SendMessage(CWM_SUBMITDONE, Dali::WUActionCheck, (LPARAM)&errors);
+        }
```

Comment:
Should use the new paradigm for checking type (as per other comment).

Reply:
@GordonSmith so sorry!